testsuite/features/step_definitions/retail_steps.rb

```diff
@@ -1,6 +1,9 @@
 # Copyright 2021-2023 SUSE LLC
 # Licensed under the terms of the MIT license.
 
+### This file contains the definitions for all steps concerning the different
```

Comment:
I would make a mention to retail functionality

Reply:
Would something like this suffice?
`This file contains the definitions for all steps concerning the different kinds of minions aswell as PXE-Boot and retail functionality.`
Probably too vague? 🤔

Reply:
I like the description you suggest.

But then the name of the file would need to change in phase 3, as it's not about Retail only anymore. Maybe `pxeboot_tests.rb`.

Reply:
But then we would have a file doing many different things again which would be ambiguously named, wouldn't we? 🤔 
Anyway, I included a mention for retail in the current header and marked the PXE-boot block as being able to maybe be split off in the future.

Reply:
ok